include/WCSimOpRaman.hh

```diff
@@ -0,0 +1,188 @@
+// * for the full disclaimer and the limitation of liability.         *
+// *                                                                  *
+// * This  code  implementation is the result of  the  scientific and *
+// * technical work of the GEANT4 collaboration.                      *
+// * By using,  copying,  modifying or  distributing the software (or *
+// * any work based  on the software)  you  agree  to acknowledge its *
+// * use  in  resulting  scientific  publications,  and indicate your *
+// * acceptance of all terms of the Geant4 Software license.          *
+// ********************************************************************
+//
+//
+// $Id: G4OpRayleigh.hh 84596 2014-10-17 07:34:41Z gcosmo $
+//
+// 
+////////////////////////////////////////////////////////////////////////
+// Optical Photon Rayleigh Scattering Class Definition
+////////////////////////////////////////////////////////////////////////
+//
+// File:        G4OpRayleigh.hh
+// Description: Discrete Process -- Rayleigh scattering of optical photons 
+// Version:     1.0
+// Created:     1996-05-31
+// Author:      Juliet Armstrong
+// Updated:     2014-08-20 allow for more material types
+//              2005-07-28 add G4ProcessType to constructor
+//              1999-10-29 add method and class descriptors
+//              1997-04-09 by Peter Gumplinger
+//              > new physics/tracking scheme
+// mail:        gum@triumf.ca
+//
+////////////////////////////////////////////////////////////////////////
+
+#ifndef WCSimOpRaman_h
+#define WCSimOpRaman_h 1
+
+/////////////
+// Includes
+/////////////
+
+#include "globals.hh"
+#include "templates.hh"
+#include "Randomize.hh"
+#include "G4ThreeVector.hh"
+#include "G4ParticleMomentum.hh"
+#include "G4Step.hh"
+#include "G4VDiscreteProcess.hh"
+#include "G4DynamicParticle.hh"
+#include "G4Material.hh"
+#include "G4OpticalPhoton.hh"
+#include "G4PhysicsTable.hh"
+#include "G4PhysicsOrderedFreeVector.hh"
+
+// Class Description:
+// Discrete Process -- Rayleigh scattering of optical photons.
+// Class inherits publicly from G4VDiscreteProcess.
+// Class Description - End:
+
+/////////////////////
+// Class Definition
+/////////////////////
+
+class WCSimOpRaman : public G4VDiscreteProcess 
+{
+
+public:
+
+        ////////////////////////////////
+        // Constructors and Destructor
+        ////////////////////////////////
+ 
+        WCSimOpRaman(const G4String& processName = "OpRaman",
+                              G4ProcessType type = fOptical);
+	~WCSimOpRaman();
+
+private:
+
+        WCSimOpRaman(const WCSimOpRaman &right);
+
+        //////////////
+        // Operators
+        //////////////
+
+        WCSimOpRaman& operator=(const WCSimOpRaman &right);
+
+public:
+
+        ////////////
+        // Methods
+        ////////////
+
+        G4bool IsApplicable(const G4ParticleDefinition& aParticleType);
+        // Returns true -> 'is applicable' only for an optical photon.
```

Comment:
```suggestion
        /// Returns true -> 'is applicable' only for an optical photon.
        G4bool IsApplicable(const G4ParticleDefinition& aParticleType);
```
Since we use doxygen, and I guess they use something bespoke, it'd be good to do this. Essentially
- move the comment line(s) above, rather than below the function
- add an extra `/` to make it a triple `///`

This is a general comment for this (and other) files. But if it's a massive pain, then don't worry